src/scintillator/loader.js

```diff
@@ -12,34 +11,33 @@ const log = debug('scintillator:loader')
 
 // TODO [#637]: Convert the `load` function to async function (instead of using `co`) in src/scintillator/loader.js
```

Comment:
When fixing an issue, it’s a good idea to remove the corresponding TODO tag. :smile: